org.moreunit.plugin/src/org/moreunit/annotation/MoreUnitAnnotationModel.java

```diff
@@ -354,4 +354,12 @@ public void removeAnnotationModelListener(IAnnotationModelListener listener)
     {
         annotationModelListeners.remove(listener);
     }
+
+    private List<MoreUnitAnnotation> copyAnnotations()
+    {
+        synchronized (annotations)
+        {
+            return new ArrayList<MoreUnitAnnotation>(annotations);

```

Comment:
you can directly return Collections.synchronizedList(annotations)

And in this case we might use them directly in the code without extracting a method